lib/index.js

```diff
@@ -21,17 +21,10 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-/**
```

Comment:
Maybe keep external/internal dependencies comment?

Reply:
We removed them all in VIP CLI, I tried to be consistent :-)